changelog.md

```diff
@@ -37,6 +37,8 @@
   unless `-d:nimLegacyHomeDir` is specified (for a transition period).
 
 ## Standard library additions and changes
+- `cuchar` now maps to `int8` instead of `char`
```

Comment:
uint8 :)
```suggestion
- `cuchar` now maps to `uint8` instead of `char`
```

Also I think it should be moved to "Changes affecting backward compatibility"

Reply:
damn you saw that before i changed it XD yes, i'll swap sections too